app/services/check_answers/section_id_finder.rb

```diff
@@ -2,9 +2,10 @@ module CheckAnswers
   class SectionIdFinder
     IDS = {
       level_of_help: "section-level-of-help-header",
+      asylum_support: "section-about_the_case-header",
       matter_type: "section-about_the_case-header",
       tribunal: "subsection-case_details-header",
-      asylum_support: "subsection-asylum_support-header",
+      # asylum_support: "subsection-asylum_support-header",
```

Comment:
I think this file is fine how it was - the value for asylum_support looks like it should be `subsection-asylum_support-header`